### PR TITLE
Feature/real time

### PR DESCRIPTION
**Pull request by CristhianCV**
socket service refactorized

### CODE REVIEW

File: src/containers/Game.svelte [hunk 1]
```diff
@@ -1,12 +1,35 @@
...
+
+  const queryString = window.location.search;
+  const urlParams = new URLSearchParams(queryString);
+  room = urlParams.get("room").toString();
+
+  game.setRoom(room);
+  game.setId();
+
+  socketService.socket.on(GAME_EVENTS.VERIFY_ROOM_AVAILABILITY, data => {
+    console.log("event on ", GAME_EVENTS.VERIFY_ROOM_AVAILABILITY);
```
Comment (JohanVillanueva) [inline src/containers/Game.svelte:19]: Remove console log

File: src/containers/Triangle.svelte [hunk 2]
```diff
@@ -3,10 +3,15 @@
   import { getContext } from "svelte";
   import { game } from "../@store";
   import Circle from "./../components/Circle.svelte";
+  import { socketService, GAME_EVENTS } from "../services/socket";
+
   const DURATION = 10;
   let types = getContext("circleTypes");
+  let socket = getContext("socket");
```
Comment (JohanVillanueva) [inline src/containers/Triangle.svelte:10]: remove unnecesary getContext for socket

File: src/@store/index.js [hunk 3]
```diff
@@ -36,7 +38,51 @@ function createGameStore() {
...
+        let { opponent, host } = game.players;
+        if (isOpponent) {
+          opponent = {
+            ...opponent,
+            id: Math.floor(Math.random() * 10000) + 1,
+          };
+        } else {
+          host = {
+            ...host,
+            id: Math.floor(Math.random() * 10000) + 1,
```
Comment (jairoederson) [inline src/@store/index.js:63]: use function to get random data

File: src/services/socket.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  EXIT_GAME: "exitGame",
+  USER_PICKED: "circleSelected",
+  GAME_RESULT_READY: "circleSelectedResponse",
+  JOIN_ROOM: "joinRoom",
+};
+
+class SocketService {
+  constructor() {
+    this.socket = null;
+    console.log("socket service is created");
```
Comment (jairoederson) [inline src/services/socket.js:18]: remove console log

**Pull request merged**